1-open-source-model.md

```diff
@@ -0,0 +1,50 @@
+
+# 1. open-source-model
+======
+Date: 18-01-2021 12:49:25
+
+We adopt an open source model of development.
+
+This means that we fully publish code, documentation, and other materials required to build,
+debug, and modify a project without needing to contact BCC.Media employees or contractors.
+
+This implicitly forces us to develop software in a way that is maintainable and does
+not rely on security-by-obscurity.
+
+## Status
+======
+Proposed
+
+## Context
+======
+
+Having a closed source model makes it really hard to involve volunteers. In the future we will need
+to expand the pool of contributors significantly if we want to keep up.
+
+In addition to that we hope that we will be able to provide inspiration and grow a community of
+developers in BCC.
+
+## Decision
+======
+
+All software development projects (unless otherwise decided for documented technical or legal reasons) should be 
+released under the [GNU AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) license.
```

Comment:
Whats the reasons for choosing this one over other licenses?

Reply:
To be honest, I didn't give it a lot of thought.

Initially I wanted to go with MIT, but then decided on GPL because it forces anyone using and modifying the software to release the patches.

What's your take here?

Reply:
After some conversation with both @andreasgan and Reng, MIT seems to be the choice.
We need to be careful about how we license the graphics. Best would be that the graphical profile is not included in the repo at all and is "sideloaded" from somewhere else at build/run time.